crates/matrix-sdk/src/authentication/oauth/mod.rs

```diff
@@ -873,7 +892,11 @@ impl OAuth {
         )
     }
 
-    /// Login via OAuth 2.0 with the Authorization Code flow.
+    /// Log in via OAuth 2.0 with the Authorization Code flow.
```

Comment:
Can we perhaps link to the section in the module level docs where we explain what the Authorization Code flow is?

Reply:
I am not sure how to do that except by cheating with a relative URL. I copied the paragraph that explains what the flow does.